### PR TITLE
Add a space on teachercon registration form

### PR DESCRIPTION
**Pull request by mehalshah**
Require ADA follow up
Require food allergy follow up
Colon on carpooling question
Update food allergy details text
Update signature on emails

### CODE REVIEW

**Comment by mehalshah** [comment]
https://trello.com/c/GTMiix33/458-third-round-of-facilitator-bug-bash-feedback

File: dashboard/app/models/pd/teachercon1819_registration.rb [hunk 1]
```diff
@@ -11,8 +11,8 @@
 #
 # Indexes
 #
-#  index_pd_teachercon1819_registrations_on_pd_application_id    (pd_application_id)
-#  index_pd_teachercon1819_registrations_on_regional_partner_id  (regional_partner_id)
+#  index_pd_summer_event1819_registrations_on_pd_application_id    (pd_application_id)
+#  index_pd_summer_event1819_registrations_on_regional_partner_id  (regional_partner_id)
```
Comment (Hamms) [inline dashboard/app/models/pd/teachercon1819_registration.rb]: ^ again

File: dashboard/app/views/pd/fit_weekend1819_registration_mailer/confirmation.html.haml [hunk 2]
```diff
@@ -8,15 +8,19 @@
...
   We noticed you responded that you’re not able to attend your assigned FiT weekend.
   Please contact us at
-  = link_to("facilitator@code.org", "mailto:facilitator@code.org")
+  = link_to("facilitators@code.org", "mailto:facilitators@code.org")
   so that we can assist you.
 
+%p
+  Thank you,
+  %br
+  Sarah
```
Comment (Hamms) [inline dashboard/app/views/pd/fit_weekend1819_registration_mailer/confirmation.html.haml:23]: I assume we don't want both "Sarah" and "Sarah Fairweather" here and below

Comment (mehalshah) [reply dashboard/app/views/pd/fit_weekend1819_registration_mailer/confirmation.html.haml:23]: We do - its by design

File: dashboard/app/models/pd/teachercon1819_registration.rb (hunk 1 above)
Comment (mehalshah) [reply dashboard/app/models/pd/teachercon1819_registration.rb]: Argh - I need to fix these indecies

**Pull request merged**